tempie.js

```diff
@@ -160,6 +162,36 @@ var _private =  {
     ], function(err) {
       callback(err, tpl);
     });
+  },
+
+  /*
+   * Finds, and registers partials for use in handlebars
+   */
+  registerPartials: function(callback) {
+    var partialsDir = path.resolve(path.join(tempie.dir, tempie.partialsDir));
+
+    fs.exists(partialsDir, function (exists) {
+      if (!exists) return callback();
+
+      parseDir(path.join(partialsDir, '**/*'), function (err, partials) {
+        if (err) return callback(err);
+
+        partials.forEach(function (partial) {
+          var dir, name;
+
+          if (partial.isDirectory) return;
+
+          dir = path.dirname(partial.filepath).replace(partialsDir, '');

```

Comment:
nope: again, README out of date:

https://github.com/firstandthird/parse-dir/blob/master/lib/parse-dir.js#L69